### ISSUE TITLE
Need to explicitly specify VirtualBox in Vagrantfile

### ISSUE THREAD

**Issue by tylerhall** [clickontyler/VirtualHostX-Issues#10]
If the user has Fusion installed, it takes preference over VirtualBox and won't boot with the image we're using.

**Comment by huebs** [comment]
The same problem occurs when Parallels is installed. Here's the error output:

```
Bringing machine 'default' up with 'parallels' provider...
==> default: Box 'ubuntu/trusty64' could not be found. Attempting to find and install...
    default: Box Provider: parallels
    default: Box Version: >= 0
==> default: Loading metadata for box 'ubuntu/trusty64'
    default: URL: https://atlas.hashicorp.com/ubuntu/trusty64
The box you're attempting to add doesn't support the provider
you requested. Please find an alternate box or use an alternate
provider. Double-check your requested provider to verify you didn't
simply misspell it.

If you're adding a box from HashiCorp's Atlas, make sure the box is
released.

Name: ubuntu/trusty64
Address: https://atlas.hashicorp.com/ubuntu/trusty64
Requested provider: [:parallels]
```

**Comment by tylerhall** [comment]
@huebs Thanks for that. Good to know.

**Comment by tylerhall** [comment]
Fixed in version 7.0.7.
